bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/GroupItem.java

```diff
@@ -239,6 +239,11 @@ protected void internalSend(Command command) {
      */
     @Override
     public State getStateAs(Class<? extends State> typeClass) {
+        // if a group does not have a function it cannot have a state

```

Comment:
not really correct - a group item without a function cannot have a _derived_ state... It could still have one itself, so returning _here_ is too early...